tests/test_timeseriesflattener/test_flattened_dataset_generation.py

```diff
@@ -0,0 +1,233 @@
+"""Integration test for the flattened dataset generation."""
+
+# pylint: disable=unused-import, redefined-outer-name
```

Comment:
```suggestion
# pylint: redefined-outer-name
```
Flake8 vil brokke sig over den samme fejl?

Reply:
Unused-import er den her weird case, hvor jeg ikke kan inline specificere det for e.g. `load_synth_outcome`, fordi `isort` og `black` konkurrerer med hinanden.

Mega-credit challenge: Check branchen ud og find en løsning ;-)

Reply:
Hmm, surprisingly så passer pre-commit hooken. Vi må lige se på det.